src/messaging/ExchangeContext.cpp

```diff
@@ -573,5 +575,31 @@ ExchangeMessageDispatch & ExchangeContext::GetMessageDispatch(bool isEphemeralEx
     return ApplicationExchangeDispatch::Instance();
 }
 
+void ExchangeContext::AbortAllOtherCommunicationOnFabric()
+{
+    if (!mSession || !mSession->IsSecureSession())
+    {
+        ChipLogError(ExchangeManager, "AbortAllOtherCommunicationOnFabric called when we don't have a PASE/CASE session");
+        return;
+    }
+
+    // Save our session so it does not actually go away.
+    Optional<SessionHandle> session = mSession.Get();
```

Comment:
It can't, because we can't get our hands on a vanilla handle: it has no copy constructor.  `Get()` returns an `Optional` and we are moving out of it here, but if we tried to get the handle itself this would not compile.

Reply:
Let's just keep it as is to get the PR in.